src/routers/TabRouter.js

```diff
@@ -38,18 +39,21 @@ export default (routeConfigs, config = {}) => {
       let state = inputState;
       if (!state) {
         const routes = order.map(routeName => {
+          const includeInitialParams = routeName === initialRouteName;
```

Comment:
Will be cleaner to do `const params = routeName === initialRouteName ? initialRouteParams : undefined;` here and then just use `params` below.

Reply:
Was unsure about letting the `params` key exist as `params: undefined`... will make the changes.